Documentation/Basics.md

```diff
@@ -1,60 +1,51 @@
-# Basic Usage #
-
-Let's say you have an endpoint, `GET https://www.example.com/comments/1`, that returns the following JSON:
+Let's say we have an endpoint, `GET https://raw.githubusercontent.com/thoughtbot/Swish/master/Documentation/example.json`
+that returns the following JSON:
 
 ```json
 {
-  "id": 1,
+  "id": 4,
   "commentText": "Pretty good. Pret-ty pre-ty pre-ty good.",
   "username": "LarryDavid"
 }
 ```
 
-We'll model this with the following struct, and implement Argo's `Decodable` protocol to tell it how to deal with JSON:
+We will model the data with the following struct
+and conform to the `Codable` protocol so we can
+turn from JSON into a Swift object.
 
 ```swift
-import Argo
-import Curry
-
-struct Comment {
+struct Comment: Codable {
   let id: Int
-  let text: String
+  let body: String
   let username: String
-}
 
-extension Comment: Decodable {
-  static func decode(_ json: JSON) -> Decoded<Comment> {
-    return curry(Comment.init)
-      <^> j <| "id"
-      <*> j <| "commentText"
-      <*> j <| "username"
+  enum CodingKeys: String, CodingKey {
+    case id = "id"
```

Comment:
FWIW, you can leave off the explicit `= "id"` if the string value is the same as the case name. (Same goes for `username` below.)